### PR TITLE
load manifest/dependencygraph/xrefmap from buildinfo

### PR DESCRIPTION
**Pull request by ansyral**

### CODE REVIEW

**Comment by ansyral** [comment]
#449

**Comment by superyyrrzz** [comment]
:+1:

File: src/Microsoft.DocAsCode.Build.Incrementals/DependencyGraph.cs [hunk 1]
```diff
@@ -59,6 +59,13 @@ public IEnumerable<string> Keys
             get { return _dictionary.Keys; }
         }
 
+        public SortedSet<string> GetDirectDependency(string key)
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.Incrementals/DependencyGraph.cs]: >GetDirectDependency [](start = 33, length = 19)

can it be resued by ReportDependency?

Comment (ansyral) [inline src/Microsoft.DocAsCode.Build.Incrementals/DependencyGraph.cs]: how?

---
In reply to: [75277009](https://github.com/dotnet/docfx/pull/618#discussion_r75277009) [](ancestors = 75277009)

Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.Incrementals/DependencyGraph.cs]: mean to reuse trygetvalue, but it's ok not reuse it

---
In reply to: [75417844](https://github.com/dotnet/docfx/pull/618#discussion_r75417844) [](ancestors = 75417844,75277009)

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**